### DISCUSSION TITLE
Local stylesheets

### DISCUSSION THREAD

**Discussion by randomllama** [blakeblackshear/frigate#5138] [Ideas]
I've introduced Frigate to my family and have inevitably ended up with recordings getting turned off due to "fat fingers".  This has been brought up before in the issues (#3539, #4206, etc.).  My proposition to work around this: include a blank stylesheet.  The user can then easily tweak the interface by hiding elements that others need not touch.

As a proof of concept, I've added this to my `/opt/frigate/web/index.html`:
`<link rel="stylesheet" href="/BASE_PATH/assets/local.css">`

The contents of `/opt/frigate/web/assets/local.css`:
`[aria-label^=Toggle] { display: none; }`

Now the toggle buttons on the main page are not displayed.  A dirty hack?  Yes.  But it's a quick and easy way to keep the technically challenged from causing problems.

**Comment by SonomaGTS** [comment]
I agree with this. At least til a means of more control over the front end can be made through the normal config. Hiding things like the toggling of the detect and recording etc
`div[aria-label^="Toggle"] { display: none; }` 
or even the config link
`a[href$="/config"] { display: none; }`
is kind of crucial when giving family members access to view the content captured by frigate.
Ideally the end game would give frigate the means of having a seperate "viewer" account family members could use.
